### ISSUE TITLE
# 20 Tensorflow 딥러닝(표 이용, 학습이 잘되는 모델 구성)

### ISSUE THREAD

**Issue by lungnahahd** [lungnahahd/Winter_Project_20-21#21]
1. layers.Dense와 layers.Activation을 분리하기
```
H = tf.keras.layers.Dense(독립변수,activation = 'swish')(X)
 // 위의 코드를 먼저 아래 코드로 분리
H = tf.keras.layers.Dense(독립변수)(X)
H = tf.keras.layers.Activation('swish')(H)
```
2. BatchNormarlization()을 분리한 코드 사이에 추가
```
H = tf.keras.layers.Dense(독립변수)(X)
H = tf.keras.layers.BatchNormalization()(H) // 해당 부분 추가하기
H = tf.keras.layers.Activation('swish')(H)
```
